network-api/networkapi/buyersguide/pagemodels/base_voting.py

```diff
@@ -6,34 +6,47 @@
 from .products.base import Product
 
 
-class ProductVote(models.Model):
-    votes = models.IntegerField(
-        default=0
-    )
+class Vote(models.Model):
```

Comment:
The only thing I did to this file is move the classes around so that they're always "X", then "BooleanX", then "RangeX", because I had to look things up in this file and the nonuniform ordering made that harder than it had to be.